apps/sentry-client-desktop/src/features/keys/HasKeys.tsx

```diff
@@ -251,29 +250,26 @@ export function HasKeys({combinedOwners, combinedLicensesMap, statusMap, isWalle
 						<PrimaryButton
 							isDisabled={selectedWallet === null}
 							onClick={copySelectedWallet}
-							className={`bg-primaryBgColor flex !h-[48px] items-center !w-[155px] text-btnPrimaryBgColor hover:bg-btnPrimaryBgColor hover:text-white text-lg uppercase font-bold !py-1 !px-[10px]`}
+							className={`flex justify-center ${selectedWallet === null ? "!h-[50px]" : "!h-[48px]"} items-center !w-[155px] text-lg uppercase font-bold !py-1 !px-[10px]`}
```

Comment:
Isn't there a `disabled:` tailwind selector for this (instead of `${selectedWallet === null ? "!h-[50px]" : "!h-[48px]"}`)?

Reply:
Changed disabled styles